src/icons/fileIcons.ts

```diff
@@ -2482,5 +2482,21 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'apps-script', fileExtensions: ['gs'] },
+    {
+      name: 'metro',
+      fileNames: [
+        'metro.config.json',
+        'metro.config.yml',
+        'metro.config.yaml',
+        'metro.config.js',
+        'metro.config.mjs',
+        'metro.config.cjs',
+        'metro.config.ts'
```

Comment:
File is not formatted via [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)

Reply:
Okay I'll format